types/react-vis/index.d.ts

```diff
@@ -0,0 +1,3868 @@
+
+        getAngle0?: (datapoint: number, info: infoObject) => void;
+
+        getColor?: (datapoint: number, info: infoObject) => void;
+
+        getColor0?: (datapoint: number, info: infoObject) => void;
+
+        getOpacity?: (datapoint: number, info: infoObject) => void;
+
+        getOpacity0?: (datapoint: number, info: infoObject) => void;
+
+        getRadius?: (datapoint: number, info: infoObject) => void;
+
+        getRadius0?: (datapoint: number, info: infoObject) => void;
+
+        getSize?: (datapoint: number, info: infoObject) => void;
+
+        getSize0?: (datapoint: number, info: infoObject) => void;
+
+        getX?: (datapoint: number, info: infoObject) => void;
+
+        getX0?: (datapoint: number, info: infoObject) => void;
+
+        getY?: (datapoint: number, info: infoObject) => void;
+
+        getY0?: (datapoint: number, info: infoObject) => void;
+
+        height?: number;
+
+        onNearestX?: (datapoint: void, info: infoObject) => void;
+
+        onNearestXY?: (datapoint: void, info: infoObject) => void;
+
+        onSeriesClick?: (datapoint: number, info: infoObject) => void;
+
+        onSeriesMouseOut?: (datapoint: number, info: infoObject) => void;
+
+        onSeriesMouseOver?: (datapoint: number, info: infoObject) => void;
+
+        onSeriesRightClick?: (datapoint: number, info: infoObject) => void;
+
+        onValueClick?: (datapoint: number, info: infoObject) => void;
+
+        onValueMouseOut?: (datapoint: number, info: infoObject) => void;
+
+        onValueMouseOver?: (datapoint: number, info: infoObject) => void;
+
+        onValueRightClick?: (datapoint: number, info: infoObject) => void;
+
+        opacityBaseValue?: number;
+
+        opacityDistance?: number;
+
+        opacityDomain?: Array<number>;
+
+        opacityRange?: Array<number>;
+
+        opacityType?: ('linear' | 'ordinal' | 'category' | 'time' | 'time-utc' | 'log' | 'literal');
+
+        radiusBaseValue?: number;
+
+        radiusDistance?: number;
+
+        radiusDomain?: Array<number>;
+
+        radiusRange?: Array<number>;
+
+        radiusType?: ('linear' | 'ordinal' | 'category' | 'time' | 'time-utc' | 'log' | 'literal');
+
+        sizeBaseValue?: number;
+
+        sizeDistance?: number;
+
+        sizeDomain?: Array<number>;
+
+        sizeRange?: Array<number>;
+
+        sizeType?: ('linear' | 'ordinal' | 'category' | 'time' | 'time-utc' | 'log' | 'literal');
+
+        stack?: boolean;
+
+        style?: Object;
+
+        width?: number;
+
+        xBaseValue?: number;
+
+        xDistance?: number;
+
+        xDomain?: Array<number | Date>;
+
+        xRange?: Array<number>;
+
+        xType?: ('linear' | 'ordinal' | 'category' | 'time' | 'time-utc' | 'log' | 'literal');
+
+        yBaseValue?: number;
+
+        yDistance?: number;
+
+        yDomain?: Array<number>;
+
+        yRange?: Array<number>;
+
+        yType?: ('linear' | 'ordinal' | 'category' | 'time' | 'time-utc' | 'log' | 'literal');
```

Comment:
A type alias for `('linear' | 'ordinal' | 'category' | 'time' | 'time-utc' | 'log' | 'literal')` would be useful.